src/api/class/instance.js

```diff
@@ -43,9 +43,9 @@ class WhatsAppInstance {
     constructor(key, allowWebhook = false, webhook = null) {
         this.key = key ? key : uuidv4()
         this.allowWebhook = config.webhookEnabled
-        if (this.allowWebhook && webhook !== null) {
+        if (this.allowWebhook && config.webhookUrl !== null) {
```

Comment:
maybe this fix that problem? #182

Reply:
i fixed custom webhook in this commit
https://github.com/salman0ansari/whatsapp-api-nodejs/commit/a5f74ac4594021d37b883af6aab65e35322a7886